theme/saas/static/css/shippable.css

```diff
@@ -142,3 +142,289 @@ blockquote {
   color: #333;
   border-left: 5px solid #ddd;
 }
+
```

Comment:
We should run this through some kind of css lint. There are few missing semi-colons.